code/game/objects/structures/tables_racks.dm

```diff
@@ -441,6 +446,11 @@
 	L.Weaken(10 SECONDS)
 	qdel(src)
 
+/obj/structure/table/glass/shove_impact(mob/living/target)
+	target.forceMove(get_turf(src))
+	table_shatter(target)
```

Comment:
Is this one required? Since `Crossed` also checks this. Will force the table to shatter when there is no gravity though